src/vstart.sh

```diff
@@ -431,7 +431,8 @@ test -d gmon && $SUDO rm -rf gmon/*
 
 
 # figure machine's ip
-HOSTNAME=`hostname -s`
+#HOSTNAME=`hostname -s`
+HOSTNAME=`hostname | awk -F "." '{print $1}'`
```

Comment:
Sorry, this shouldn't be there... it was from my lab at home. I will take it out.